base/notify-celery-email-send/celery-email-send-deployment.yaml

```diff
@@ -21,6 +21,37 @@ spec:
       labels:
         app: celery-email-send
     spec:
+      initContainers:
+      - name: wait-cwagent-ready
+        env:
+          - name: STATSD_HOST
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        image: api
+        imagePullPolicy: IfNotPresent
+        command: ["sh", "-c"]
```

Comment:
Any advantage over having the scripts inlined rather than executing a file? Because that was the quick solution as the existing scripts are in the API repo?

Reply:
Last week it didn't occur to me to just use the api image, and I didn't want to build a new image for a single thing...

We can move this to scripts, it will just require two PRs instead of one. You're probably right in terms of keeping things consistent.